sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -632,20 +639,36 @@ private static void AddHttpOutputBinding(IList<ExpandoObject> bindingMetadata, s
         private static void AddExtensionInfo(IDictionary<string, string> extensions, CustomAttribute attribute)
         {
             AssemblyDefinition extensionAssemblyDefinition = attribute.AttributeType.Resolve().Module.Assembly;
+            TryAddExtensionInfo(extensions, extensionAssemblyDefinition);
+        }
 
+        private static bool TryAddExtensionInfo(IDictionary<string, string> extensions, AssemblyDefinition extensionAssemblyDefinition, bool usedByFunction = true)
+        {
             foreach (var assemblyAttribute in extensionAssemblyDefinition.CustomAttributes)
             {
                 if (string.Equals(assemblyAttribute.AttributeType.FullName, Constants.ExtensionsInformationType, StringComparison.Ordinal))
                 {
                     string extensionName = assemblyAttribute.ConstructorArguments[0].Value.ToString();
-                    string extensionVersion = assemblyAttribute.ConstructorArguments[1].Value.ToString();
+                    string extensionVersion = assemblyAttribute.ConstructorArguments[1].Value.ToString();                    
+                    bool implicitlyRegister = false;
 
-                    extensions[extensionName] = extensionVersion;
+                    if (assemblyAttribute.ConstructorArguments.Count >= 3)
```

Comment:
Question: Why 3?

Reply:
That would force us to load the assembly during the build -- which can be problematic with versioning, etc. That's why we're trying to only figure out the values by using the constructor params, which are able to be determined via assembly metadata. Because we own that piece, we know it's the value we're looking for.